docs/operate/cli.mdx

```diff
@@ -62,6 +60,25 @@ sudo mv restate /usr/local/bin/
 ```
 
 </CH.Code>
+
+Check if the CLI is configured correctly:
+
+```shell
+restate whoami
+```
+
+Print help to see all available commands:
+
+```shell
+restate --help
+```
+
+<Admonition type="tip">
+    The CLI can be a useful tool for debugging and troubleshooting.
+    Have a look at the [introspection page](/operate/introspection) for a list of useful commands .
+</Admonition>
+
+
```

Comment:
I think the list of env variables is incomplete: https://github.com/restatedev/restate/blob/main/cli/src/cli_env.rs. Do we want to specify them as a kind of reference somewhere or is it good enough to only list he most important ones? Note things like `RESTATE_EDITOR` can only be found by looking at the code.

Reply:
We seem to refer to this page from the introspection page: https://c94c7840.documentation-beg.pages.dev/operate/introspection?interface=cli#to-edit-application-state.

Reply:
Can I just link to here https://github.com/restatedev/restate/blob/main/cli/src/cli_env.rs in a note and say that the full list of variables can be found there? It's a bit much to list all of them